cattle/utils.py

```diff
@@ -73,6 +78,42 @@ def unwrap(json_object):
         return json_object
 
 
+class CadvisorAPIClient(object):
+    def __init__(self, host, port, version='v1.2', proto='http://'):
+        self.url = '{0}{1}:{2}/api/{3}'.format(proto, host, str(port), version)
```

Comment:
You don't need to change this, but for future reference, you shouldnt need str(port). The format function will take care of conversion to a string.